streaming_form_data/_parser.pyx

```diff
@@ -229,6 +232,7 @@ cdef class _Parser:
                     name = params.get('name')
                     if name:
                         part = self._part_for(name) or self.default_part
+                        part.headers_parsed(params)
```

Comment:
What do you think about having a `filename` attribute instead in the `Part` class? And then setting it by reading the `filename` from the `Content-Disposition` header value?

Reply:
Sounds good.

I have limited experience with python and I never used attributes before. This is why I missed this option.